vgm.sh

```diff
@@ -32,7 +38,7 @@ module-whatis "ALICE Modulefile for $PKGNAME $PKGVERSION-@@PKGREVISION@$PKGHASH@
 # Dependencies
 module load BASE/1.0 GEANT4/$GEANT4_VERSION-$GEANT4_REVISION ROOT/$ROOT_VERSION-$ROOT_REVISION
 # Our environment
-setenv VGM_ROOT \$::env(BASEDIR)/$PKGNAME/\$version
-prepend-path LD_LIBRARY_PATH \$::env(VGM_ROOT)/lib
-$([[ ${ARCHITECTURE:0:3} == osx ]] && echo "prepend-path DYLD_LIBRARY_PATH \$::env(VGM_ROOT)/lib")
+set VGM_ROOT \$::env(BASEDIR)/$PKGNAME/\$version
+setenv VGM_ROOT \$VGM_ROOT
```

Comment:
Do you really need the $VGM_ROOT environment?

Reply:
@ktf , the recipes in the packages defaults (xyz.sh) were taken from master; I did not write them myself and when updating I always just modified what was needed for correct functioning of the updated version. If you have simplifications and/or clean-ups, I suggest to propose them in a PR in master, where they would get also tested and woul be then propagated to other tags.

Reply:
$VGM_ROOT  is used in 02.sh